packages/aws-cdk/README.md

```diff
@@ -255,6 +255,11 @@ $ cdk doctor
   - AWS_SDK_LOAD_CONFIG = 1
 ```
 
+### MFA support
+
+If `mfa_serial` is found in the active profile of the shared ini file AWS CDK
+will ask for token defined in the `mfa_serial`. This token will be provided to STS assume role call.
+
```

Comment:
perhaps also add an example of how/where users can configure it by showing a sample profile and entry of the serial

Reply:
I've added example profile config that uses mfa_serial